lighty-modules/lighty-restconf-nb-community/src/main/java/io/lighty/modules/northbound/restconf/community/impl/CommunityRestConf.java

```diff
@@ -10,14 +10,11 @@
 import com.google.common.base.Stopwatch;
 import com.google.common.base.Throwables;
 import io.lighty.core.controller.api.AbstractLightyModule;
-import io.lighty.modules.northbound.restconf.community.impl.config.JsonRestConfServiceType;
```

Comment:
When you look into `import io.lighty.modules.northbound.restconf.community.impl.config.JsonRestConfServiceType;
` you can see that `DRAFT_02` is still used.

Please remove `JsonRestConfServiceType` completely.

Reply:
OK